.github/workflows/runner.yml

```
@@ -41,7 +41,7 @@ jobs:
 
   windows-build:
     if: ${{ vars.DISABLE_GODOT_CI == '' }}
-    name: 🏁 Windows
+    name: 🪟 Windows
```

I prefer the flag personally.
```suggestion
    name: 🏁 Windows
```

---

I didn't know about that! Thanks, Aaron!